Aquality.Selenium/tests/Aquality.Selenium.Tests/Resources/settings.azure.json

```diff
@@ -52,6 +52,30 @@
       "options": {},
       "startArguments": []
     },
+    "edge": {
+      "webDriverVersion": "82.0.458.0",
```

Comment:
do we need webDriverVersion here?

Reply:
I can remove it, it just a full example of settings file and now it is just dead weight, but if we want to run our tests on different browsers it could be useful.

Reply:
This line you've added has no effect as we don't use webDriverVersion for Edge browser type.
So existence of this property make the user think we support it while actually we're not.